.github/pull_request_template.md

```diff
@@ -1,14 +1,15 @@
-Thank you for your contribution to the iTwinUi-react project! Please make sure to complete all of the items below before submitting the PR.
+<!--
+Thank you for your contribution to the iTwinUI-react project!
+Please describe your PR here and make sure to complete all of the items below before submitting.
+-->
+
+Closes issue #
 
 ## Checklist
```

Comment:
Maybe add that issue is linked. Just to be more explicit.

Reply:
I don't think it's possible for anyone without write access to link issues using the UI. That's why I added this "Closes" keyword.